sab2/Scenes/Home Screen/Home TableView/HomeScreenViewController.swift

```diff
@@ -0,0 +1,121 @@
+    var listPresenter: ListPresenter?
+    var newsAdapter = NewsAdapter()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        _ = self.navigationController?.navigationBar
+        titleNavigationBarItem()
+        leftNavigationBarItem()
+        rightNavigationBarItem()
+        
+        listPresenter?.loadResponse()
+//        listPresenter?.loadSlider()
+        listPresenter?.loadVideos()
+        listPresenter?.loadImage()
+        listPresenter?.loadArticles()
+        
+        // datasource and delegate in adapter
+        self.homeTableView.delegate = newsAdapter
+        self.homeTableView.dataSource = newsAdapter
+        registerCells()
+        homeTableView.estimatedRowHeight = 145
+        homeTableView.rowHeight = UITableView.automaticDimension
+        newsAdapter.reloadData = reloadData
+    }
+    
+    func titleNavigationBarItem() {
+        let logImageView = UIImageView(frame: CGRect(x: 0, y: 0, width: 44, height: 27))
+        logImageView.contentMode = .scaleAspectFit
+        let logoImage = #imageLiteral(resourceName: "logo")
+        logImageView.image = logoImage
+        navigationItem.titleView = logImageView
+    }
+    
+    func leftNavigationBarItem() {
+        let containView = UIView(frame: CGRect(x: 0, y: 0, width: 17, height: 21))
+        _ = UIView(frame: CGRect(x: 0, y: 0, width: 17, height: 21))
+        let notificationIconImageview = UIImageView(frame: CGRect(x: 0, y: 0, width: 17, height: 21))
+        notificationIconImageview.image = #imageLiteral(resourceName: "notification-icon")
+        notificationIconImageview.contentMode = UIView.ContentMode.scaleAspectFit
+        notificationIconImageview.layer.cornerRadius = 10
+        notificationIconImageview.layer.masksToBounds = true
+        containView.addSubview( notificationIconImageview)
+        let leftBarButton = UIBarButtonItem(customView: containView)
+        self.navigationItem.leftBarButtonItem = leftBarButton
+        
+    }
+    
+    func rightNavigationBarItem() {
+        let containView = UIView(frame: CGRect(x: 0, y: 0, width: 25, height: 25))
+        let imageview = UIImageView(frame: CGRect(x: 0, y: 0, width: 25, height: 25))
+        imageview.image = #imageLiteral(resourceName: "user")
+        imageview.contentMode = UIView.ContentMode.scaleAspectFit
+        imageview.layer.cornerRadius = 20
+        imageview.layer.masksToBounds = true
+        containView.addSubview(imageview)
+        let rightBarButton = UIBarButtonItem(
+            image: #imageLiteral(resourceName: "user"),
+            style: .plain,
+            target: self,
+            action: nil)
+     
+        self.navigationItem.rightBarButtonItem = rightBarButton
+        
+    }
+    
+    func registerCells() {
+        let defaultCellNib = UINib(nibName: "HomeTableViewCell", bundle: nil)
+        homeTableView.register(defaultCellNib, forCellReuseIdentifier: "HomeTableViewCell")
```

Comment:
Using strings in code **"HomeTableViewCell"** is a bad behavior 
try to find a way to not use them as a string in code